channels/disp/client/disp_main.c

```diff
@@ -176,22 +176,26 @@ UINT disp_send_display_control_monitor_layout_pdu(DISP_CHANNEL_CALLBACK* callbac
 UINT disp_recv_display_control_caps_pdu(DISP_CHANNEL_CALLBACK* callback, wStream* s)
 {
 	DISP_PLUGIN* disp;
+	DispClientContext *context;
+	UINT ret = CHANNEL_RC_OK;
 
 	disp = (DISP_PLUGIN*) callback->plugin;
+	context = (DispClientContext *)disp->iface.pInterface;
 
 	if (Stream_GetRemainingLength(s) < 12)
 	{
-		WLog_ERR(TAG, "not enough remaining data");
+		WLog_ERR(TAG, "%s: not enough remaining data", __FUNCTION__);
```

Comment:
Actually the function name can be added to the WLog format string via env variable